### ISSUE TITLE
Wildcard currently results in double the metric collection

### ISSUE THREAD

**Issue by tiffanyfj** [intelsdi-x/snap-plugin-collector-docker#23]
E.g. if you have two containers and you are collecting metrics with *, it will collect the metric 4x. After the PR on snap for allowing you to use anything to fill for a wildcard goes through, this should be fixed.

**Comment by IzabellaRaulin** [comment]
This issue was related with exposing by docker collector plugin in previous version the same metric **in two variants**: with an asterisk representing docker_id and also with docker_id explicitly, e.g.:
```
/intel/docker/*/stats/cgroups/memory_stats/stats/total_active_anon
/intel/docker/*/stats/cgroups/memory_stats/stats/total_active_file
/intel/docker/a186b5421e69/stats/cgroups/memory_stats/stats/total_active_anon
/intel/docker/a186b5421e69/stats/cgroups/memory_stats/stats/total_active_file
```
As the result, when in task manifest the metric "/intel/docker/\*/stats/cgroups/memory_stats/stats/total_active_anon" is requested, that means that metric "/intel/docker/a186b5421e69/stats/cgroups/memory_stats/stats/total_active_anon" has been collected two times.

Exposing in metric catalog metric with an asterisk and also metric represented explicitly which is included in the first one is a wrong way, because causes multiplication in collection process.

To sum up, this issue was resolved by https://github.com/intelsdi-x/snap-plugin-collector-docker/pull/32

Metric Catalog - BEFORE:
```
/intel/docker/*/stats/cgroups/memory_stats/stats/total_active_anon
/intel/docker/*/stats/cgroups/memory_stats/stats/total_active_file
/intel/docker/a186b5421e69/stats/cgroups/memory_stats/stats/total_active_anon
/intel/docker/a186b5421e69/stats/cgroups/memory_stats/stats/total_active_file
```

Metric Catalog - AFTER:
```
/intel/docker/*/stats/cgroups/memory_stats/stats/total_active_anon
/intel/docker/*/stats/cgroups/memory_stats/stats/total_active_file
```
